qa-tests-backend/src/test/groovy/K8sRbacTest.groovy

```diff
@@ -246,7 +246,7 @@ class K8sRbacTest extends BaseSpecification {
         withRetry(45, 2) {
             def stackroxBindings = RbacService.getRoleBindings()
             def orchestratorBindings = orchestrator.getRoleBindings() + orchestrator.getClusterRoleBindings()
-            assert stackroxBindings.size() == orchestratorBindings.size()
+            assert stackroxBindings.size() == orchestratorBindings.size(), "Binding sizes differ"
```

Comment:
Add a comment to prevent default message with a list of all bindings that brings noise.